### ISSUE TITLE
Box 2d with temperature boundary conditions ?

### ISSUE THREAD

**Issue by paugier** [exabl/snek5000#27]
I'd like to create a 2d solver with such boundary conditions:

```
cbox.rea
-2                      spatial dimension
2                     number of fields
#========================================================
#
#    This gives a 9 x 1 box, periodic in x,
#    here used for Rayleigh Benard convection.
#
#    Note that number of fields < 0 implies that box.rea
#    will be ascii.
#
#========================================================
#
Box
-16    -16            nelx,nely,nelz for Box)
0      1     1      x0 x1 ratio
0      1     1      y0 y1 ratio
W  ,W  ,W  ,W  ,    V bc's  ! NB:  3 characters each !
t  ,t  ,I  ,I  ,    T bc's  !      You must have 2 spaces!!
```

`info_box` does not support this (https://github.com/exabl/snek5000/blob/master/src/snek5000/operators.py#L381).

What would be a good way to support that ?

**Comment by ashwinvis** [comment]
I can do that :+1:

**Comment by paugier** [comment]
What about the dimension (2D) ? Is it already supported somehow ?

**Comment by ashwinvis** [comment]
I am writing a test case for the 2D case now... hold on.

**Comment by ashwinvis** [comment]
Do you have an example of the SIZE file. I wonder if that needs any modification.

**Comment by ashwinvis** [comment]
@paugier I am happy with the changes in branch `fix/temp-bcs` #28. Let me know if there is anything I missed.
